hera_cal/redcal.py

```diff
@@ -1178,8 +1232,10 @@ def redcal_run(input_data, filetype='uvh5', firstcal_ext='.first.calfits', omnic
             this altitude, calibration is skipped and the integrations are flagged.
         flag_nchan_low: integer number of channels at the low frequency end of the band to always flag (default 0)
         flag_nchan_high: integer number of channels at the high frequency end of the band to always flag (default 0)
-        conv_crit: maximum allowed relative change in omnical solutions for convergence
-        maxiter: maximum number of omnical iterations allowed before it gives up
+        fc_conv_crit: maximum allowed changed in firstcal phases for convergence
+        fc_maxiter: maximum number of firstcal iterations allowed for finding per-antenna phases
+        oc_conv_crit: maximum allowed relative change in omnical solutions for convergence
+        oc_maxiter: maximum number of omnical iterations allowed before it gives up
```

Comment:
Crazy idea: could parameter docstrings be built algorithmically to avoid having these same strings show up in a file 30 times?

Reply:
Sadly, no.